Plate_1_data/1.wave2_data/README.md

```diff
@@ -0,0 +1,32 @@
+# ASC Bleedthrough Assessment - Wave 2
```

Comment:
May also consider adding a verbal description of what this means and the overall goal of this assessment.

Reply:
Added a sentence with quick description here. Good point!